docs/DeveloperGuide.md

```diff
@@ -682,7 +682,7 @@ with phone number, using a prefix to identify them.
 * needs to keep track of the daily sales
 * prefers desktop apps over other types
 * prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* is reasonably comfortable using command-line interface (CLI) apps
```

Comment:
No need to write the long form here, I have added the long form in the introduction!

Reply:
Okay thanks